src/hardware/sblaster.cpp

```diff
@@ -2288,6 +2303,11 @@ class SBLASTER final {
 		} break;
 		}
 
+		cms = find_cms();
```

Comment:
"find" kinda makes sense for `sbtype` which is an enum, but `is_cms_enabled` is probably a better name here.